.changeset/little-files-notice.md

```diff
@@ -0,0 +1,6 @@
+---
+"@skeletonlabs/skeleton": patch
+---
+
+chore: add framework dependency sources
```

Comment:
Not sure what this entails. And other than the extra bullet point on the changelog won't affect the version if we have multiple changelogs.